release_notes.md

```diff
@@ -1,118 +1,69 @@
+- `Known To Be Used In Ransomware Campaigns`
 
-#### Widget Additions
+### NIST NVD Connector Action Isolation
 
-We’ve integrated cutting-edge widgets to execute your playbooks with a single click and track the execution with an intuitive, user-friendly wizard.
+The **NIST NVD Connector** action, responsible for fetching CVE details, has been isolated into a dedicated playbook named **Update CVE Details from NIST**. This playbook includes the setting `Ignore Error` set to *Yes* to handle intermittent API issues more gracefully.  
 
-- Playbook Execution Wizard
-- Playbook Buttons
+The **Update CVE Details from NIST** playbook is referenced as the final step in the **Find Known Exploited Vulnerabilities (KEV) CVEs** playbook. This change addresses the intermittent reliability issues with the NIST NVD APIs and ensures uninterrupted processing of CVE data.
 
-#### Picklist Enhancements
+### Key Store Enhancements
 
-The following picklists and the containing items were renamed to provide clearer context and more specific terminology, reflecting its focus on outbreak-related alerts. This change enhances user understanding and aligns with the outbreak management system's terminology, making it easier for analysts to identify and manage relevant outbreak alerts.
+- Key Store records associated with Outbreaks are now tagged with **`Outbreak Alert`**
 
-- The picklist *Rule Type* has been renamed to **Threat Hunt Rule Type**.
+### Schedule Changes for Solution Pack Deployment
 
-- The picklist *Record Status* has been renamed to **Outbreak Alert Status**
-    - This picklist has additional options - `New` and `Deactivate` to clearly indicate outbreak alerts' statuses.
-    - The picklist item *Active* has been renamed to **Tracking** to mark that an outbreak alert is ready to investigate.
+The crontab schedule for **Outbreak_Automated-Deployment-Outbreak-Alert-Response-Solution-Pack** has been adjusted to run **every 3 hours**. This ensures the timely deployment of new outbreak alert solution packs.  
 
-#### New Key Store Records
+These solution packs, created upon receiving outbreak alerts from FortiGuard, include YARA rules, Sigma rules, and other detection mechanisms designed to facilitate the hunting and identification of outbreaks within a network.
 
-- The addition of the following Record Sets for use by the Configuration Wizard imparts a modular structure to threat hunt integrations and automatic installation of outbreak-specific response solution packs:
-    - `outbreak-threat-hunt-tools`
-    - `outbreak-threat-hunt-tools-params`
-    - `outbreak-auto-install-time-frame`
+### Playbook Enhancements and Cleanups
 
-- Effortlessly manage your threat hunt integrations. Each integration and its precise hunt parameters are stored in the `outbreak-threat-hunting-workflow-config` key store record, ensuring a streamlined and efficient threat-hunting process.
+- The **Investigate Outbreak** playbook has been optimized to remove redundancy.
```

Comment:
Please add one more release note
Earlier only max 5000 IOC records were fetched and investigated, Now All the IOCs linked with Outbreak Alert will be fetched and investigated